.env.test

```diff
@@ -0,0 +1,6 @@
+# define your env variables for the test env here
+KERNEL_CLASS='App\Kernel'
+APP_SECRET='$ecretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=999999
+PANTHER_APP_ENV=panther
+DATABASE_URL=postgres://postgres@127.0.0.1:5432/postgres
```

Comment:
perhaps postgres://db_user@127.0.0.1:5432/db_name just to discourage the casual use of the 'postgres' administrative user